### DISCUSSION TITLE
[IntelliSense] Allow enabling TailwindCSS IntelliSense without a tailwind.config.js file

### DISCUSSION THREAD

**Discussion by matthewmueller** [tailwindlabs/tailwindcss#12315] [Ideas]
Hi there 👋 

I'm working on a Tailwind-compatible compiler for the Go language. I was hoping to be able to use the Intellisense plugin for VSCode, but I noticed:

> In order for the extension to activate you must have tailwindcss installed and a Tailwind config file named tailwind.config.{js,cjs,mjs,ts} in your workspace.

In this case, I'd like users to not be required to create a tailwind.config.js file, rather it's an optional enhancement. I was wondering if you could provide a setting that just always enables it? Perhaps with some internal, basic default config? This is also similar to how Typescript's VSCode plugin works, where `tsconfig.json` is an optional, helpful enhancement.

I could also see this being useful for users that just want to get up & running with an index.html and Tailwind inside their editor, without fussing around with NPM and config files.

Thanks!

**Comment by OtavioGonzaga** [comment]
I'm having the same issue trying to use tailwind cdn on Yew (Rust).
